### ISSUE TITLE
ZynVoice button does not announce engine descriptions

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#1669]
# Describe the issue
When adding an instrument chain (and probably other types), the engine selector view shows the categories, engines and info. The info is not accessible to blind users.
# To Reproduce
## Steps to reproduce the behavior:
1. Enable ZynVoice.
2. Add an instrument chain.
3. In the engine selector view, press the zynvoice button.
## Expected behaviour
Anouncment of view, category, engine and info.
## Actual behaviour
Anouncment of view, category, engine only.
# Additional context
Ideally the star rating, knob complexity and description should be announced.

# Configuration
## Hardware
```
Raspberry Pi 5 Model B Rev 1.0
Audio: Tascam US-16x08
Display: MIPI DSI 800x480 (inverted)
Wiring: TOUCH_ONLY
I2C: Not detected
Profile: v5
```
## System
```
Debian GNU/Linux 12 (bookworm)
Build Date: 2024-10-02
Memory: 36% (2952M/8063M)
Internal Storage: 36% (79G/234G ext4)
Temperature: 57.1ºC
Overclock: Maximum
```
## MIDI & UI
```
Tuning: 440 Hz
Master Channel: Off
Preload Presets: On
ZS3 (SubSnapShots): On
Power Save: 10 minutes
Audio Levels on Snapshots: Off
```
## Software
```
zyncoder: vangelis (a4b603f) 
zynthian-ui: vangelis (75ac3f4)
zynthian-sys: vangelis (5103faa)
zynthian-data: vangelis (3f50b18)
zynthian-webconf: vangelis (6eb3717)
```
